Docs/Workflows.md

```diff
@@ -99,3 +99,24 @@ jobs:
 If this is for a submodule of [Lombiq's Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/), the `repo` input can be omitted, because the above is its default value. Otherwise, use your parent repository's address in the `{owner}/{repo_name}` format.
 
 Refer to [Github Actions reusable workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#overview) for more information.
+
+## Jira issue-creating for community activities workflow
+
+Creates Jira issues for community activities happening on GitHub, like issues, discussions, pull requests being opened.
+
+Set up secrets for the `JIRA_` parameters as explained [here](https://github.com/marketplace/actions/jira-login#enviroment-variables). You may use secret names without the `DEFAULT_` prefix, but that's our recommendation for organization-level secrets, so you have defaults but can override them on a per-repository basis.
+
+The secrets with the `_JIRA_ISSUE_DESCRIPTION` suffix should contain templates for the Jira issues to be created, using the internal markup format of Jira (not Markdown). Example for one for `ISSUE_JIRA_ISSUE_DESCRIPTION`:
```

Comment:
Shouldn't this be `JIRA_*`, `DEFAULT_*` and `*_JIRA_ISSUE_DESCRIPTION`? Especially the latter looks weird to me as it currently is.